### PR TITLE
Added 1.16.0 support

### PR DESCRIPTION
**Pull request by Chitpixel**
Haven't done other testing, but the specter is able to join the game without issues now.

### CODE REVIEW

File: plugin.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: "0.9"
 author: Falkirks
 main: specter\Specter
 api: ["3.11"]
-mcpe-protocol: [390]
+mcpe-protocol: [390, 407]
```
Comment (thebigsmileXD) [inline plugin.yml:6]: Maybe we should just bump this to one version for simplicity, though the patch will work on both versions

File: src/specter/network/SpecterInterface.php [hunk 2]
```diff
@@ -211,6 +211,7 @@ public function decodeAdditional()
             $pk->clientId = 1;
             $pk->xuid = "xuid here";
             $pk->identityPublicKey = "key here";
+            $pk->clientData["SkinResourcePatch"] = base64_encode('{"geometry": {"default": "geometry.humanoid.custom"}}');
```
[the remark this answers is not in the document]
Comment (Chitpixel) [reply src/specter/network/SpecterInterface.php:214]: Yes, this is what I got when I dumped what my client sent. 
Check this out: https://github.com/pmmp/PocketMine-MP/blob/e105578be0728065a14a0cb8f7d438f6114c5711/src/pocketmine/network/mcpe/protocol/types/LegacySkinAdapter.php#L38-L52

[the remark this answers is not in the document]
Comment (Chitpixel) [reply src/specter/network/SpecterInterface.php:214]: Should I json_encode an array (how PocketMine does it in that code) or is this fine?

[the remark this answers is not in the document]
Comment (thebigsmileXD) [reply src/specter/network/SpecterInterface.php:214]: Ah thanks, looks fine, i rechecked

**Pull request merged**